parm/met_config/GridStatConfig_wrapped

```diff
@@ -148,7 +148,7 @@ nbrhd = {
    field      = BOTH;
    shape      = ${NEIGHBORHOOD_SHAPE};
    width      = [ ${NEIGHBORHOOD_WIDTH} ];
-   cov_thresh = [ >=0.5 ];
+   ${NEIGHBORHOOD_COV_THRESH}
```

Comment:
George, it looks like we're expecting:
${NEIGHBORHOOD_COV_THRESH} to resolve to "cov_thresh = [ >=0.5 ];"

Why do we set it up that way here, whereas on the previous line, we expect:
${NEIGHBORHOOD_WIDTH} to resolve to "3, 5, 7, 9" without the lefthand side before the equal sign?

How do you pick which whether the environment variable should or should not include the LHS of the equal sign?

Reply:
@JohnHalleyGotway , specifying the variable and value is the new paradigm we have adopted to avoid keeping default values in sync with the MET config files. Ideally we would update all of the wrappers to handle them this way. For the sake of getting this functionality available more quickly, I decided to add the new variable with the new approach and held off on updating the rest.